03-Cartographic-Modelling-with-Forest-Inventories.Rmd

```diff
@@ -0,0 +1,796 @@
+data that we are retrieving. We will use this principle and expand the
+SQL statement above in the next step in order to retrieve only the tiny
+fraction of the VRI data that we need.
+
+**Step 4:** From the top ribbon in ArcGIS Pro, navigate to the "Map"
+tab, then click the down arrow under "Add Data" and select "Query
+Layer..." from the drop-down menu. In the "New Query Layer" dialogue
+window that appears, click the "Datasource" drop-down menu at the top
+and you should see the UBC PostgreSQL server connection to the `vri`
+database listed there that you added at the beginning of this task. It
+should look something like
+"PostgreSQL-FRST-PostgreSQL-vri(student).sde".
+
+If not, then click the little database connection button next to the
+drop-down menu in the dialogue window to add the database connection.
+(Note: in this specific case, you will need to add the port number to
+the path string like `FRST-PostgreSQL.ead.ubc.ca,5432`)
+
+We are going to write a select query to retrieve these data from the
+server, but we do not need all the columns, nor all the rows. We only
+need polygons that are managed forests and for those polygons, we only
+need a few attributes that we are going to use later in the lab
+("bec_zone_code", "bec_subzone", and "proj_age_1").
+
+**Step 5:** Toggle on "List of Tables" to see the table names and then
+click on the VRI table in the list "vri.public.vancouver_island_vri" to
+show the list of columns. (If you get an error message here, then save
+your project, restart ArcGIS Pro and try again). We are going to write a
+SQL query to select the polygons that meet some of our initial criteria
+and then return the values of the columns (attributes) that we will need
+later. In the "Name" field, name the new query layer
+"vancouver_island_vri_select".
+
+```{r 03-arcgispro-query-layer-1, out.width= "75%", echo = FALSE}
+    knitr::include_graphics("images/03-arcgispro-query-layer-1.png")
+```
+
+The VRI data include descriptions about both forested and non-forested
+lands in British Columbia. Since we are only concerned with forests, we
+are going to only find polygons that are managed forests. We can do this
+simply with
+`SELECT * FROM vancouver_island_vri WHERE for_mgmt_land_base_ind = 'Y'`.
+See here that we have not defined any specific column names to return,
+`SELECT *` means *give me all the columns*, so in this example, we are
+only subsetting on the rows with `WHERE for_mgmt_land_base_ind = 'Y'`,
+but not the columns. If instead we used
+`SELECT for_mgmt_land_base_ind FROM vancouver_island_vri`, then we would
+get all the polygon values for only the `for_mgmt_land_base_ind` column
+name (subset columns, but not rows). You can test this SQL query but do
+not complete it as a new layer.
+
+#### Q1. What does for_mgmt_land_base_ind select for? What parameters are required to receive a "Yes" value? Hint: take a look at the VRI Relational Data Dictionary.
```

Comment:
Here, and for the other questions, add the point value in brackets after the question eg `(10 points)`

Reply:
Also, there is no report, so allocate the 5 points for formatting to the map.